src/common/proxy_wasm/ngx_proxy_wasm_properties.c

```diff
@@ -877,25 +879,58 @@ ngx_proxy_wasm_properties_get_host(ngx_proxy_wasm_ctx_t *pwctx,
 
     hpn = (host_props_node_t *)
               ngx_str_rbtree_lookup(&pwctx->host_props_tree, path, hash);
+    if (!hpn) {
```

Comment:
```suggestion
    if (hpn == NULL) {
```

Reply:
This as well (already updated locally, but then started noticing more; is the PR currently lacking the reviews changes?)

Reply:
Or did those changes never made it? Since I already started updating the PR locally please refrain from making more updates, unless it is indeed missing a large set of review changes...

Reply:
ohh I might have reset the branch too far back in time?... I'm pretty sure I renamed the `negative_cache` stuff. Let me look it up.

Reply:
@thibaultcha Yes, something on my end went out of sync (my local checkout shows `origin` for this branch as containing the updated code, which it clearly doesn't).

To simplify things and not get in the way of your merge, I've pushed the commit with the applied reviews in a separate branch: https://github.com/Kong/ngx_wasm_module/tree/feat/ffi-properties-2

I've also triggered a CI run: https://github.com/Kong/ngx_wasm_module/actions/runs/6934996957

Edit: for ease of merge I also pushed a branch with the code reviews in a separate commit, which I found locally: 
[feat/ffi-properties-code-review](https://github.com/Kong/ngx_wasm_module/tree/feat/ffi-properties-code-review)